components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/Exceptions.kt

```diff
@@ -7,28 +7,28 @@ package mozilla.components.service.fxa
 /**
  * High-level exception class for the exceptions thrown in the Rust library.
  */
-typealias FxaException = mozilla.appservices.fxaclient.FxaException
+typealias FxaException = mozilla.appservices.fxaclient.FxaErrorException
```

Comment:
Name change due to current limitations of bindings generator, which has strong opinions about the name of exception classes.